frontend/benefit/applicant/src/components/applications/forms/application/step2/ApplicationFormStep2.tsx

```diff
@@ -244,7 +244,6 @@ const ApplicationFormStep2: React.FC<DynamicFormStepComponentProps> = ({
                 }
                 options={subsidyOptions}
                 id={fields.paySubsidyPercent.name}
-                name={fields.paySubsidyPercent.name}
```

Comment:
This was the problem here: https://github.com/City-of-Helsinki/yjdh/runs/4986525433?check_suite_focus=true
It seems that `name` was not necessary